Lib/http/server.py

```diff
@@ -132,7 +144,7 @@ class ThreadingHTTPServer(socketserver.ThreadingMixIn, HTTPServer):
 class HTTPSServer(HTTPServer):
     def __init__(self, server_address, RequestHandlerClass,
                  bind_and_activate=True, *, certfile, keyfile=None,
-                 password=None, alpn_protocols=None):
+                 password=None, alpn_protocols=None, response_headers=None):
```

Comment:
```suggestion
                 password=None, alpn_protocols=None, **http_server_kwargs):
```

And pass `http_server_kwargs` to `super()`